src/themes/saas/components/Account/Create.tsx

```diff
@@ -0,0 +1,370 @@
+import {
```

Comment:
not sure what is this component and why is here, does this do the same as signup? Or is another workflow?

Reply:
And is correct to put this inside of themes folder?

Reply:
It is correct if the view requires a complete different layout for one theme compared to the other. But this specific view might need to be loaded when BACKEND_URL is defined, rather than given the theme saas.

Reply:
It's another component https://github.com/vocdoni/interoperability/issues/201

Reply:
I will do this